component/index.android.js

```diff
@@ -41,6 +41,13 @@ NotificationsComponent.prototype.scheduleLocalNotification = function(details: O
 	RNPushNotification.scheduleLocalNotification(details);
 };
 
+NotificationsComponent.prototype.setApplicationIconBadgeNumber = function(number: number) {
+       if (!RNPushNotification.setApplicationIconBadgeNumber) {
+               return;
+       }
```

Comment:
This check is for backwards compatibility. We use CodePush to push javascript changes ahead of native (Play store) updates, and this lets us do that.